### PR TITLE
update icon: msdos (original, plain, line)

### PR DESCRIPTION
**Pull request by canaleal**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR is related to this issue #1977**

## Link to prove your SVG is correct and up-to-date.

- [MSDOS wikipedia](https://commons.wikimedia.org/wiki/File:Msdos-icon.svg)

It's not the official link, but I didn't change the icon from the previous PR.
- https://github.com/devicons/devicon/pull/726

I updated the color to `#000` as that is the most common color in the original logo, and it's already used in the line svg.

### CODE REVIEW

**Review by lunatic-fox — DISMISSED** [review]
Problem solved! 👍🏼
Thank you for this contribution! 🚀 

What do you think about remove the padding of those icons @canaleal and @Snailedlt?

**Review by Snailedlt — COMMENTED** [review]
Looks good, but like @lunatic-fox mentioned the padding should be removed :)

File: icons/msdos/msdos-plain.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...972 0h20.09v3.969h8.063v4.208h4.208V20.33h-16.241v3.73h8.183v3.85h3.85v3.85h4.208v12.511h-4.208v4.214h-3.85v3.85H79.882v-3.85h-4.209v-4.214h-3.85V31.76h11.908v-3.368h-8.058v-3.85h-3.85V8.183h3.85V3.97h8.059V0ZM8.058 0h16.121v3.969h4.214v8.182h3.969v7.699h3.85v8.059h3.61V19.85h4.209v-7.699h3.85V3.97h3.85V0h16.725v52.333H51.729V36.57h-3.366v7.699h-3.85v8.063H31.758V44.27h-3.849v-7.699h-3.73v15.763H7.818V0Zm0 0"/><path d="M31.758 76.272H24.18v27.55h11.549v-7.698H31.76zm36.091 0h-3.61v23.462h3.61V95.64h3.97v-3.969h8.062v-3.366h-4.214v-3.85h-3.85v-3.85h-3.969zm35.733 19.852h-7.34v3.61h7.34zM4.208 47.88h40.305v3.969h3.366V47.88h28.273v3.969h3.73V47.88h28.387v3.969h8.063v3.85h3.85v4.452h3.969v12.272h-3.97v3.85H108.27v3.73h8.063v3.968h3.85v3.85h3.969v28.273h-3.97v3.969h-3.85v3.968h-8.062V128H83.732v-3.969H80.36V128H51.729v-3.969h-7.215V128H3.85V47.88Zm103.464 27.909v-3.366h-4.089v-4.334h-3.49v7.699zm0 0"/><path d="M60.15 4.209H64v43.91h-3.85ZM51.969 20.09h3.974v4.209h-3.974ZM20.091 4.209h3.85v4.214h-3.85zm3.85 8.182h4.208v3.73h-4.21zm4.208 7.699h3.974v4.209H28.15zm3.974 8.059h3.85v3.85h-3.851Zm3.85 8.182h4.089v3.73h-4.089Zm-19.732-20.21V48.12h3.85V24.3h3.85v-4.21h-3.85v-3.969zm0 0"/></svg>
```
Comment (Snailedlt) [inline icons/msdos/msdos-plain.svg:1]: This one has multiple paths :O
Should be unified

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Size looks good now. I now noticed that msdos-plain has multiple paths though, so those need to be unified

**Review by Snailedlt — APPROVED** [review]
Everything looks good now! Great work as always! 💯 
image
image
image

**Review by lunatic-fox — APPROVED** [review]
Great one! 👍🏼
Thank you for this contribution! 🚀

> *PS: This icon is really nostalgic.*

**Pull request merged**